pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -37,27 +49,35 @@ async def run(self):
                 raise ValueError("Advisory status not in REL_PREP yet ...")
             if advisory.errata_state == "SHIPPED_LIVE":
                 self._logger.info("Advisory status already in SHIPPED_LIVE, update subtask 9 ...")
-                self._jira_client.complete_subtask(self.parent_jira_key, 8, "Advisory status already in SHIPPED_LIVE")
+                self._jira_client.complete_subtask(self.parent_jira_key, 8, "Advisory status already in SHIPPED_LIVE", self.force)
             self._logger.info("Advisory status already in post REL_PREP, update subtask 7 ...")
-            self._jira_client.complete_subtask(self.parent_jira_key, 6, "Advisory status already in REL_PREP")
+            self._jira_client.complete_subtask(self.parent_jira_key, 6, "Advisory status already in REL_PREP", self.force)
 
-            sdk_build = [b for b in sum(list(map(list, advisory.errata_builds.values())),[]) if b.startswith('openshift-enterprise-operator-sdk-container')]
+            sdk_build = [b for b in sum(list(map(list, advisory.errata_builds.values())), []) if b.startswith('openshift-enterprise-operator-sdk-container')]
             if not sdk_build:
                 self._logger.info("No SDK build to ship, update subtask 8 then close ...")
-                self._jira_client.complete_subtask(self.parent_jira_key, 7, f"No SDK build to ship, operator_sdk_sync job: {self.runtime.get_job_run_url()}")
+                self._jira_client.complete_subtask(self.parent_jira_key, 7, f"No SDK build to ship, operator_sdk_sync job: {self.runtime.get_job_run_url()}", self.force)
                 return
+            # check if sync sdk subtask already done
+            parent_jira = self._jira_client.get_issue(self.parent_jira_key)
+            sdk_subtask = self._jira_client.get_issue(parent_jira.fields.subtasks[7].key)
+            if sdk_subtask.fields.status.name == "Closed":
+                # check if already synced
+                output = subprocess.getoutput(f"curl -s {constants.OPERATOR_MIRROR_URL}/{self.assembly}/ |grep 'File not found'")
```

Comment:
do a pre-check if currently assembly already exists on the mirror, if already synced the following sync step can be skipped